### PR TITLE
feat(UI): List users roles

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This adds a modal that list users roles for a dashboard.

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [x] 23.10.x (master)

<h2> How this pull request can be tested ? </h2>

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by bdauria — APPROVED** [review]

File: centreon/lang/de_DE.UTF-8/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -15478,3 +15478,12 @@ msgstr ""
 
 # msgid "If you leave {{dashboardName}}, your changes will not be saved."
 # msgstr ""
+
+# msgid "Dashboard access rights"
+# msgstr ""
+
+# msgid "Users roles"
```
Comment (smau-centreon) [inline centreon/lang/de_DE.UTF-8/LC_MESSAGES/messages.po:15485]:
```suggestion
# msgid "User roles"
```

File: centreon/lang/pt_BR.UTF-8/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -16277,3 +16277,12 @@ msgstr ""
 
 # msgid "If you leave {{dashboardName}}, your changes will not be saved."
 # msgstr ""
+
+# msgid "Dashboard access rights"
+# msgstr ""
+
+# msgid "Users roles"
```
Comment (smau-centreon) [inline centreon/lang/pt_BR.UTF-8/LC_MESSAGES/messages.po:16284]:
```suggestion
# msgid "User roles"
```

File: centreon/lang/pt_PT.UTF-8/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -16267,3 +16267,12 @@ msgstr "Você não salvou as alterações, deseja continuar?"
 
 # msgid "If you leave {{dashboardName}}, your changes will not be saved."
 # msgstr ""
+
+# msgid "Dashboard access rights"
+# msgstr ""
+
+# msgid "Users roles"
```
Comment (smau-centreon) [inline centreon/lang/pt_PT.UTF-8/LC_MESSAGES/messages.po:16274]:
```suggestion
# msgid "User roles"
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

File: centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -17680,7 +17680,7 @@ msgid "Dashboard access rights"
 msgstr "Droits d'accès au tableau de bord"
 
 msgid "Users roles"
-msgstr "Rôles utilisateurs"
+msgstr "Rôle utilisateurs"
```
Comment (smau-centreon) [inline centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:17683]:
```suggestion
msgstr "Rôles utilisateur"
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

**Review by smau-centreon — APPROVED** [review]

**Review by g10 — APPROVED** [review]
LGTM the 2 minor details can be addressed later

File: centreon/packages/ui/src/components/List/ItemText.tsx [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import { ListItemText } from '@mui/material';
```
[the remark this answers is not in the document]
Comment (Thebarda) [reply centreon/packages/ui/src/components/List/ItemText.tsx]: Done

**Review by g10 — APPROVED** [review]

**Pull request merged**